internal/logger/logger.go

```diff
@@ -130,7 +132,7 @@ func (l *FunLogger) Loading(format string, a ...any) {
 	defer l.Wg.Done()
 	message := fmt.Sprintf(format, a...)
 	// if running in a non-interactive terminal, don't print the loading animation
-	if !isInteractiveTerminal() && isCILogs() {
+	if !isInteractiveTerminal() && l.isCILogs() {
```

Comment:
Question: Is `isCILogs` ever used alone? Does it make sense to be queried in `isInteractiveTerminal()` instead?

Reply:
no, is just used for this case, you are right